src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/EventMap.cs

```diff
@@ -161,20 +161,21 @@ public readonly void RaiseEvent<TArg>(Action<TEventHandler, TArg> invoker, TArg
             // will raise that as a fatal exception, but OperationCancelledExceptions might be able to propagate through and fault the task we are using in the
             // chain. I'm choosing to use ReportWithoutCrashAndPropagate, because if our theory here is correct, it seems the first exception isn't actually
             // causing crashes, and so if it turns out this is a very common situation I don't want to make a often-benign situation fatal.
```

Comment:
This comment is stale now. This looks like it was written by me, but since the comment has a link to a query (rather than the actual bug), it's quite useless.

Reply:
Killed the comment, things are different enough now that it's really not applicable.